localbuild.ps1

```diff
@@ -29,17 +27,32 @@ if ($Setup) {
     }
 }
 
-if ($Platform -like "all") {
-    foreach ($Plat in $Platforms) {
-        foreach ($Config in $Configurations) {
-            Write-Host "Building $Plat $Config..."
-            & ".\scripts\build.ps1" -SourcesPath $SourcesPath -OutputPath $OutputPath -Platform $Plat -Configuration $Config
+if ($Configuration -eq "all") {

```

Comment:
Suggestion: Use `ValidateSet`, and allow an explicit list of architectures as opposed to "special" metavalues.

https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-7#attributes-of-parameters

This also removes arg-validated boilerplate code.

Reply:
Cool, I'm still a PowerShell newb :)

---
In reply to: [468250255](https://github.com/microsoft/v8-jsi/pull/20#discussion_r468250255) [](ancestors = 468250255)

Reply:
Aren't we all? :)

Still, if there's hurry to merge this, we can always make incremental upgrades. The script seems correct to me.